packages/uikit-react-native-foundation/src/components/ProgressBar/index.tsx

```diff
@@ -42,6 +42,15 @@ const ProgressBar = ({ current = 100, total = 100, trackColor, barColor, overlay
     return NOOP;
   }, [percent]);
 
+  const progressBarPosition = (() => {
```

Comment:
The progress bar that displays time should always be positioned on the left.
Therefore, when `doLeftAndRightSwapInRTL` is true, specifying `right: 0` will position it on the left.